tools/virt-builder/about.md

```diff
@@ -8,13 +8,13 @@ description: Tool to quickly build (and customize) virtual machine images
 # Virt-builder
 
 [Virt-builder](http://libguestfs.org/virt-builder.1.html) is a tool for
-quickly building new virtual machines. You can build a variety of VMs
-for local or cloud use, usually within a few minutes or less.
-Virt-builder also has many ways to customize these VMs. Everything is
+quickly building new virtual machines.  You can build a variety of
+VMs for local or cloud use, usually within a few minutes or less.
+Virt-builder also has many ways to customize these VMs.  Everything is
```

Comment:
Note: this can be all in a single line, as jekyll generator does its own web page wrapping. Also multiple spaces do not matter too afaik.
See https://developer.fedoraproject.org/tools/virt-builder/about.html.

Reply:
Are you saying, don't wrap lines and just keep typing? It's just the muscle memory -- also, if I open a file in `vi`or `emacs`, I'd prefer the lines to be wrapped properly at 72 columns.  The two spaces thing is also just muscle memory :-)